OG/OG/AST/SemanticError.cs

```diff
@@ -0,0 +1,9 @@
+namespace OG.gen
+{
+    public class SemanticError
+    {
+        public int Line;
+        public int Column;
+        public string Msg;
```

Comment:
Overvej at give dem en {get; set;}

Igen, bare for best practice.

Og lav gerne en public override string ToString() {} metode, så man bare kan kalde SemanticError objektet, uden at skulle konstruere alle mulige sætninger andre steder i programmet.

Reply:
Jeps